test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -65,10 +70,12 @@
 	private static final String FILE_SEPARATOR = System.getProperty("file.separator");
 	private static final String USER_DIR = System.getProperty("user.dir");
 	private static final String OS_NAME_PROPERTY = System.getProperty("os.name");
+	private static String vmargsJarFilename;
```

Comment:
It's generally better to group like kinds of variables together.  So the `static final` fields would be grouped together and the mutable ones would be a separate grouping.  It makes it easier to find them when reading the code.

Reply:
I was missing the "final".